src/visualization/client/Turi Create Visualization/src/user_interface/src/elements/Annotate/InfiniteScroll/ImageContainer/index.js

```diff
@@ -49,7 +45,8 @@ class ImageContainer extends Component {
              onClick={this.image_click.bind(this)}>
           <img src={this.props.src.src}
              className={`${style.ImageContainerSource} ${style.ImageContainerSourceSelected}`}
-             style={this.resizeImage(this.props.src.width, this.props.src.height)}/>
+             style={this.resizeImage(this.props.src.width, this.props.src.height)}
+	     alt="Can't Find"/>
```

Comment:
Let's leave the alt text blank (empty string), since there isn't a good alternate explanation for the contents of the image -- I don't think "Can't Find" is quite correct, since for a user with a screen reader, the image is found, it's just that they can't see it.